hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/SpanEventViewGenerator.java

```diff
@@ -292,10 +293,18 @@ String getRequestUrl(Event event, Protocol protocol) {
     switch (protocol) {
       case PROTOCOL_HTTP:
       case PROTOCOL_HTTPS:
-        return EnrichedSpanUtils.getFullHttpUrl(event).orElse(event.getHttp().getRequest().getPath());
+        return getHttpUrl(event).orElse(null);
```

Comment:
This can be simplified with:
```Optional.ofNullable(event.getHttp())
        .map(ai.traceable.platform.datamodel.eventfields.http.Http::getRequest)
        .map(Request::getUrl);```